peerconnection.go

```diff
@@ -992,6 +992,29 @@ func (pc *PeerConnection) SetRemoteDescription(desc SessionDescription) error {
 			}
 
 			t, localTransceivers = findByMid(midValue, localTransceivers)
+			// Fix simulcast tracks removing
+			if t != nil && t.Receiver() != nil && len(t.Receiver().Tracks()) > 1 {
+				switch direction {
+				case RTPTransceiverDirectionInactive:
+					if err := t.Stop(); err != nil {
+						return err
+					}
+				case RTPTransceiverDirectionRecvonly:
+					if recv := t.Receiver(); recv != nil {
+						if err := recv.Stop(); err != nil {
+							return err
+						}
+					}
+				case RTPTransceiverDirectionSendonly, RTPTransceiverDirectionSendrecv:
+					t.setDirection(RTPTransceiverDirectionRecvonly)
+					receiver, err := pc.api.NewRTPReceiver(kind, pc.dtlsTransport)
+					if err != nil {
+						return err
+					}
+					t.setReceiver(receiver)
+				}
+				continue
+			}
 			if t == nil {
 				t, localTransceivers = satisfyTypeAndDirection(kind, direction, localTransceivers)
 			} else if direction == RTPTransceiverDirectionInactive {
```

Comment:
is this still necessary?

Reply:
I don't think so! if the test still passes we are all good :)

Reply:
If this is removed this test fails `TestPeerConnection_Renegotiation_DisableTrack`